### ISSUE TITLE
Retrieve tasks/incident from GLPI (Helpdesk)

### ISSUE THREAD

**Issue by rverchere** [kanboard/kanboard#1579]
Hello,

We use GLPI (http://www.glpi-project.org/) for the IT service desk, with incidents and requests from our employees.
I'm starting using the kanban & kanboard tool to manage my IT team (internal, business projects & co), but not want to do twice the work to enter tasks from the support/helpdesk.

So I would like to have a plugin that retrieve or modify GLPI tasks automatically.

Thanks for your work, it's a great & simple kanban board!

**Comment by bdrayer** [comment]
would be very interesting to mix both world.

I actually use GLPI for inventory and I use Kanboard for ticketing because it's more user friendly and other users use it for the rest of tasks management of the company.

It would be great to make a plugin that can synchronize both solutions but there will be a huge work because many fields would need to be added on Kanboard side.

I will try during my holidays to add some fields on Kanboard to fit my needs but synchronization with GLPI will need more programming skills that I have.

If someone with the necessary programming skills is interested to do the job, I will be happy to help with the design (I know both solutions and what fields need to be synchronized) and documentation.

**Comment by rverchere** [comment]
@bdrayer, I would love to know how you use Kanboard for ticketing. How many users? Difference between incidents, requests? Etc.

I can also help to test and give some feedback, but I'm not enough a good developer to create such a plugin from scratch, only a kind of sysadmin with scripting skills.

**Comment by bdrayer** [comment]
@rvechere, we are a small company (about 40 users) and I am the only IT guy. I have tried to do other products for ticketing but nobody, but me, use it. GLPI is really interesting for ticketing but the kanboard view is more useful for me and other people are using kanboard for other things inside the company so they are more lickly to use it to fill a ticket.

At the beginning I have made 2 different projects for change request, and incidents management but it's confusing for users so I now only have 1 project for everything. 

I think kanboard is not ready for such use if you are a "big" company, with many IT guy and need to manage hundred tickets a day but it cover my needs. The only downside is that, for now, I can't link it to a resource (the computer with Serial Number XYZ for example) or create a ticket on a behalf of an other person.

I have the same problem as you, I am a sysadmin with scripting skills but not programming skills (I understand PHP but have difficulty to write it, particularly when you use object programming, ...)
